src/main/java/io/nats/client/impl/NatsJetStream.java

```diff
@@ -247,7 +247,7 @@ JetStreamSubscription createSubscription(String subject,
 
             userCC = so.getConsumerConfiguration();
 
-            validateNotSupplied(userCC.getMaxPullWaiting(), ConsumerConfiguration.CcNumeric.MAX_PULL_WAITING.initial(), JsSubPushCantHaveMaxPullWaiting);
+            validateNotSupplied(userCC.getMaxPullWaiting(), ConsumerConfiguration.CcChangeHelper.MAX_PULL_WAITING.initial(), JsSubPushCantHaveMaxPullWaiting);
```

Comment:
CcChangeHelper is commented as 
> INTERNAL CLASS ONLY, SUBJECT TO CHANGE

The name was changed because I needed to support AckWait b/c the server populates ack wait in the consumer info response if the user does not supply it.

Reply:
not quite sure what you mean - server will set a default if not set.

Reply:
This is where I track the defaults that the server sets and returns in the json.